experiments/iroh-swift-ffi-spike/rust/src/lib.rs

```
@@ -0,0 +1,438 @@
+        "endpoint": {
+            "type": "peer",
+            "id": endpoint.endpoint.id().to_string(),
+            "direct_addrs": direct_addrs,
+            "relay_url": relay_url,
+        },
+        "priority": 5,
+    });
+    string_to_c(route.to_string())
+}
+
+/// Waits until the endpoint has a home relay connection (so dial-by-id from
```

Clippy (1.96.0)

[warning] unsafe function's docs are missing a `# Safety` section

(warning)